packages/core/src/index.ts

```diff
@@ -487,6 +500,10 @@ function resolveModule<T>(cur: T | null, path: string): T | null {
   try {
     return require(require.resolve(path))
   } catch (e) {
+    if (!e.message.includes('Cannot find module')) {
```

Comment:
I don't know, but may be is it `e.statusCode` exist for this situation?

Reply:
It's `e.code: 'MODULE_NOT_FOUND'`